docs/index.rst

```diff
@@ -12,6 +12,7 @@ A Python package for the analysis of correlations of light curves and their stat
 
    recipes/mutis_workflow.ipynb
    recipes/Synthethic_generation_of_light_curves.ipynb
+   recipes/Flare_analysis.ipynb
```

Comment:
just saying.. in general I'd always use alphabetic ordering in lists, in large lists I've found it useful for folder content comparisons and searching for orphans